Framework/Algorithms/src/MonteCarloAbsorption.cpp

```diff
@@ -184,381 +206,57 @@ void MonteCarloAbsorption::exec() {
       } else {
         // elastic case already initialized
       }
-      doSimulation(detPos, lambdaIn, lambdaOut, yValues[bin], eValues[bin]);
+      std::tie(signal[j], std::ignore) =
```

Comment:
It is at the moment for consistency with the other absorption algorithms that do nothing with errors. It was unclear to me what to do with the errors here when interpolate. Should they stay the same or not. Do you have an opinion on that?